hathor/builder/cli_builder.py

```diff
@@ -300,13 +301,34 @@ def create_manager(self, reactor: Reactor) -> HathorManager:
 
         cpu_mining_service = CpuMiningService()
 
+        # We are remodeling the whitelist_only trope.
+        # Now, we must use solely the p2p_whitelist flag, which shall receive
+
+        # Check whitelist pathing. Default values:
+        p2p_wl = self._args.x_p2p_whitelist or 'default'
+        p2p_wl = p2p_wl.strip()
+        p2p_whitelist: URLPeersWhitelist | FilePeersWhitelist | None = None
+
+        # If in testnet, it should always be disabled.
+        testnet = self._args.testnet
```

Comment:
Why? The default would be enabled only for mainnet. Any other network would be disabled by default but it could be override by the command line.